docs/rule/kotlin_library.soy

```diff
@@ -0,0 +1,188 @@
+{namespace kotlin_library}
+
+/***/
+{template .soyweb}
+  {call buck.page}
+    {param title: 'kotlin_library()' /}
+    {param navid: 'rule_kotlin_library' /}
+    {param prettify: true /}
+    {param description}
+      A rule that is used to define a set of Kotlin files that can be compiled
+      together.
+    {/param}
+    {param content}
+
+{call buck.rule}
+{param status: 'FROZEN' /}
```

Comment:
We don't usually mark new rules as frozen since we might be changing them as we flesh them out.